serverless/src/main.rs

```diff
@@ -44,7 +67,76 @@ pub fn main() {
         stats_flusher,
     });
 
-    if let Err(e) = mini_agent.start_mini_agent() {
-        error!("Error when starting serverless trace mini agent: {e}");
+    tokio::spawn(async move {
+        let res = mini_agent.start_mini_agent().await;
+        if let Err(e) = res {
+            error!("Error when starting serverless trace mini agent: {e:?}");
+        }
+    });
+
+    let mut metrics_flusher = if dd_use_dogstatsd {
+        let metrics_aggr = Arc::new(Mutex::new(
+            MetricsAggregator::new(Vec::new(), CONTEXTS)
+                .expect("Failed to create metrics aggregator"),
+        ));
+
+        info!("Starting dogstatsd");
```

Comment:
Should this be `debug` logs instead of `info?`

Reply:
Changed this and the `Starting serverless trace mini agent` log to both be `debug` logs.